### PR TITLE
op-guide: add guide of using docker

### PR DESCRIPTION
**Pull request by iamxy**
@siddontang @shenli @qiuyesuifeng

### CODE REVIEW

File: op-guide/docker-across-multiple-hosts.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  pingcap/tikv \
+  --addr="0.0.0.0:20160" \
+  --advertise-addr="${host3}:20160" \
+  --store="/tidata/tikv3" \
+  --dsn=raftkv \
+  --pd="${host1}:2379,${host2}:2379,${host3}:2379" \
+  --cluster-id=1
+```
+
+## 任意一台主机上启动 TiKV
```
Comment (shenli) [inline op-guide/docker-across-multiple-hosts.md]: tikv-> tidb

File: op-guide/docker-across-multiple-hosts.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  pingcap/tidb \
+  --store=tikv \
+  --path="${host1}:2379,${host2}:2379,${host3}:2379?cluster=1" \
+  -L warn
+```
+
+## 使用 MySQL 客户端连接 TiDB 进行测试
+
+```bash
+mysql -h ${host1} -P 4000 -u root -D test
```
Comment (shenli) [inline op-guide/docker-across-multiple-hosts.md]: TiDB may be not on host1

File: op-guide/docker.md [hunk 3]
```diff
@@ -21,130 +23,357 @@ docker pull pingcap/tikv
...
-docker run -d --name tidb \
-    -p 4000:4000 \
-    -v /etc/localtime:/etc/localtime:ro \
-    pingcap/tidb:latest \
-    -L info \
-    --store=tikv \
-    --path="${host_ip1}:2379?cluster=1" \
-    -P 4000
+
+#### 使用客户端记性测试
```
Comment (shenli) [inline op-guide/docker.md]: "记性测试" -> "进行测试"

**Comment by iamxy** [comment]
PTAL

**Comment by shenli** [comment]
LGTM
@qiuyesuifeng @siddontang PTAL

File: op-guide/docker-across-multiple-hosts.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+|Name|Host IP|
+|----|-------|
+|**host1**|192.168.1.100|
+|**host2**|192.168.1.101|
+|**host3**|192.168.1.102|
+
+每台机器已经安装最新版本 Docker, 并且拉取了最新的 TiDB/TiKV/PD 的镜像。
+
+``bash
```
Comment (qiuyesuifeng) [inline op-guide/docker-across-multiple-hosts.md]:
```bash ?

File: op-guide/docker-across-multiple-hosts.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+  pingcap/tikv \
+  --addr="0.0.0.0:20160" \
+  --advertise-addr="${host3}:20160" \
+  --store="/tidata/tikv3" \
+  --dsn=raftkv \
+  --pd="${host1}:2379,${host2}:2379,${host3}:2379" \
+  --cluster-id=1
+```
+
+## 4. **host1** 上启动 TiDB
```
Comment (qiuyesuifeng) [inline op-guide/docker-across-multiple-hosts.md]: 在 **host1** 上启动 TiDB？

**Comment by iamxy** [comment]
PTAL @qiuyesuifeng

File: op-guide/docker.md [hunk 6]
```diff
@@ -21,130 +23,354 @@ docker pull pingcap/tikv
...
+```bash
+mysql -h 127.0.0.1 -P 4000 -u root -D test
 ```
+
+## 使用 `docker-compose`, 更方便地构建集群
+
+只需要一个简单的 `docker-compose.yml`:
+
+```bash
+version: '2'
```
Comment (qiuyesuifeng) [inline op-guide/docker.md]: Why using version 2 here?

File: op-guide/docker.md [hunk 7]
```diff
@@ -21,130 +23,354 @@ docker pull pingcap/tikv
...
+      - --name=pd2 
+      - --client-urls=http://0.0.0.0:2379
+      - --peer-urls=http://0.0.0.0:2380
+      - --advertise-client-urls=http://pd2:2379
+      - --advertise-peer-urls=http://pd2:2380
+      - --initial-cluster=pd1=http://pd1:2380,pd2=http://pd2:2380,pd3=http://pd3:2380
+      
+    privileged: true
+
+  pd3:
```
Comment (qiuyesuifeng) [inline op-guide/docker.md]: How about distinguish the `pd3` name and `--name=pd3`?

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**